pkgs/tools/admin/pulumi/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "pulumi-${version}";
+  version = "v0.16.2";
```

```suggestion
  version = "0.16.2";
```

>  The version part of the name attribute _must_ start with a digit (following a dash)

From https://nixos.org/nixpkgs/manual/#sec-package-naming

Then use `v${version}` in the URLs